Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -87,6 +92,28 @@ def get_events(self, config_ids: str, offset: str | None = '', limit: str | int
             new_offset = str(from_epoch)
         return events, new_offset
 
+    def get_events_with_offset(
+        self,
+        config_ids: str,
+        offset: str | None = '',
+        limit: str | int | None = None,
+        from_epoch: str | None = ''
```

Comment:
api supports to also, lets add support for this

Reply:
But this is only used for fetch-events, not for manual command.
For manual command we do support to_epoch (see as an example that we don't support to_epoch in fetch alerts)